tests/test_critical_dependencies_from_recorded_activities.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
General: do the assertion error messages live up to the standard of OpenCLSim?

Reply:
I assume assert in pytest is accepted/desirable...